backend/task/data_from_fire.py

```diff
@@ -0,0 +1,188 @@
+"""
+@author: Scarlett Zhang
+This file has 1 class:
+DataFromFire: data pipeline for fire data.
+"""
+import os
+import logging
+from datetime import datetime
+import datetime
+import rootpath
+import psycopg2
+from paths import FIRE_DATA_DIR
+from typing import Tuple, List
+import time
+
+
+from backend.task.runnable import Runnable
+from backend.data_preparation.crawler.fire_crawler import FireCrawler, FireEvent
+from backend.data_preparation.extractor.fire_extractor import FireExtractor, IncompleteShapefileError
+from backend.data_preparation.dumper.fire_dumper import FireDumper
+
+rootpath.append()
+logger = logging.getLogger('TaskManager')
+
+
+class DataFromFireRunnable(Runnable):
+    """
+    Run once per day/week/, not any time
+    """
+    def __init__(self):
+        """
+        Initialize runnable object
+        """
+        # crawler object initialization: can add more state names to crawl
+        self.crawler = FireCrawler(["California"])
+        # extractor object initialization
+        self.extractor = FireExtractor()
+        # dumper object initialization
+        self.dumper = FireDumper()
+        self.start_time = 2018
```

Comment:
can this be an argument that take in from TaskManager when calling this runnable?